src/wagtail_personalisation/templates/modeladmin/wagtail_personalisation/segment/dashboard.html

```diff
@@ -38,11 +38,11 @@ <h2>{{ segment }}</h2>
                                                 <li class="stat_card">
                                                     {% trans "This segment is Static" %}
                                                     <span class="icon icon-fa-user">
-                                                        {{ segment.sessions.count|localize }}
-                                                        {% if segment.sessions.count < segment.count %}
+                                                        {{ segment.static_users.count }}
```

Comment:
It's probably good to have the `|localize` here. I know the admin is only in english but I think it's good habit

Reply:
👌